### PR TITLE
Implemented "RpcHttpBodyOnly" capability

### PR DESCRIPTION
**Pull request by AnatoliB**
Resolving issue https://github.com/Azure/azure-functions-powershell-worker/issues/292.

The goal is to reduce data duplication in function invocation requests, which will reduce the I/O volume between the host and the PS worker, and potentially allow larger request payloads.

### CODE REVIEW

File: src/Utility/TypeExtensions.cs [hunk 1]
```diff
@@ -99,6 +90,21 @@ internal static object ToObject(this TypedData data)
...
+            switch (rpcHttpBody.DataCase)
+            {
+                case TypedData.DataOneofCase.String:
+                    return rpcHttpBody.String;
+
+                case TypedData.DataOneofCase.Bytes:
+                    return Encoding.UTF8.GetString(rpcHttpBody.Bytes.ToByteArray());
+
+                default:
+                    return rpcHttpBody.ToObject();
```
Comment (Francisco-Gamino) [inline src/Utility/TypeExtensions.cs]: For the default case in which ```rpcHttpBody``` is neither a string nor bytes, then both ```httpRequestContext.Body``` and ```httpRequestContext.RawBody```  will be rpcHttp.Body.ToObject(). Is this intentional?

Comment (AnatoliB) [reply src/Utility/TypeExtensions.cs]: As far as I understand, the host will be sending us either `TypedData.DataOneofCase.String` or `TypedData.DataOneofCase.Bytes` now, so the default branch will not be executed. However, we need to come up with a safe default behavior if the host sends anything else in the future. Invoking `ToObject()` seems to be a reasonable default to me, but I'm open to other suggestions.

Also, don't pay too much attention to the resulting `httpRequestContext.Body` now. This PR focuses on `RawBody` only, and the fix for https://github.com/Azure/azure-functions-powershell-worker/issues/241 is also coming soon, in a separate PR.

Comment (Francisco-Gamino) [reply src/Utility/TypeExtensions.cs]: I thought about it a bit more. I was thinking that we should keep the same behavior as other workers. However, this has not been implemented in other workers.

Would it be possible to add a TODO there to check how other workers will handle this? Other than that, LGTM

Comment (AnatoliB) [reply src/Utility/TypeExtensions.cs]: This is in fact implemented in the NodeJS worker (see [fromRpcHttpBody](https://github.com/Azure/azure-functions-nodejs-worker/blob/d6a0c97ef7b0142a5c737c5b80d939da060bb189/src/converters/RpcHttpConverters.ts#L40)), and it seems to be doing the same.

Comment (Francisco-Gamino) [reply src/Utility/TypeExtensions.cs]: Great. Thanks for checking.

**Pull request merged**